### PR TITLE
Settings Alerts

### PR DESCRIPTION
**Pull request by tsheng1**
### Added

- Added alerts when saving settings
- Add indenting changes

### CODE REVIEW

**Review by btalma — APPROVED** [review]

File: react/Admin.tsx [hunk 1]
```diff
@@ -109,212 +138,260 @@ const ShipStationAdmin: FC = () => {
...
+            <Alert 
+              type="success" 
+              onClose={() => {
+                setSettingsState({
+                  ...settingsState,
+                  warehousesComplete: false
+                })
+              }}
+            >
+              Warehouses created successfully.
```
Comment (wender) [inline react/Admin.tsx:391]: Use FormattedMessage instead

File: react/Admin.tsx [hunk 2]
```diff
@@ -109,212 +138,260 @@ const ShipStationAdmin: FC = () => {
...
+            <Alert 
+              type="success" 
+              onClose={() => {
+                setSettingsState({
+                  ...settingsState,
+                  hooksComplete: false
+                })
+              }}
+            >
+              Webhook created successfully.
```
Comment (wender) [inline react/Admin.tsx:365]: Use FormattedMessage instead

File: react/Admin.tsx [hunk 3]
```diff
@@ -109,212 +138,260 @@ const ShipStationAdmin: FC = () => {
...
+            <Alert 
+              type="success" 
+              onClose={() => {
+                setSettingsState({
+                  ...settingsState,
+                  saveComplete: false
+                })
+              }}
+            >
+              Settings saved successfully.
```
Comment (wender) [inline react/Admin.tsx:338]: Use FormattedMessage instead

**Review by wender — CHANGES_REQUESTED** [review]
Label attributes should also use formatMessage with translatable keys, I know it was not there before, but we need them to be translatable

**Review by wender — APPROVED** [review]

**Pull request merged**